storage/src/adb/operation.rs

```diff
@@ -145,13 +146,17 @@ impl<K: Array, V: Array> Operation<K, V> {
 }
 
 impl<K: Array, V: Array> Codec for Operation<K, V> {
-    fn write(&self, writer: &mut impl Writer) {
-        writer.write_fixed(&self.data);
+    fn write(&self, buf: &mut impl BufMut) {
+        assert!(self.data.len() == Self::LEN_ENCODED);
+        buf.put(&self.data[..]);
     }
 
-    fn read(reader: &mut impl Reader) -> Result<Self, CodecError> {
-        let value = reader.read_n_bytes(Self::LEN_ENCODED)?;
-        Self::try_from(value.as_ref()).map_err(|e| CodecError::Wrapped("Operation", e.into()))
+    fn read(buf: &mut impl Buf) -> Result<Self, CodecError> {
+        commonware_codec::util::at_least(buf, Self::LEN_ENCODED)?;
```

Comment:
is there a reason you are using the fully-qualified path here?

Reply:
since it's a one-off, I thought it was slightly more readable than pure `at_least` with no context, or a `util::at_least`. 

Modified slightly to `CodecUtils::at_least` after importing `utils` as `CodecUtils`